lib/sandbox/pmapi.js

```diff
@@ -142,9 +143,14 @@ Postman = function Postman (bridge, execution, onRequest) {
             return _.isFunction(callback) && callback.call(self, new Error('sendrequest: nothing to request'));
         }
 
-        onRequest(PostmanRequest.isRequest(req) ? req : (new PostmanRequest(req)), function (err, resp) {
+        onRequest(PostmanRequest.isRequest(req) ? req : (new PostmanRequest(req)), function (err, resp, history) {
+            history &&
```

Comment:
minor point: this fits better in an if-block because it has multiple check followed by assignment and no use of the expression boolean.
```
if (history && !PostmanCookieList.isCookieList(history.cookies)) {
    history.cookies = new PostmanCookieList({}, history.cookies);
}
```